alpine/alpine-infra/src/main/java/alpine/persistence/AlpineQueryManager.java

```diff
@@ -343,6 +343,17 @@ public OidcUser addUserToTeams(final OidcUser user, final List<String> teamNames
         });
     }
 
+    /**
+     * Returns a complete list of all subclasses extending User.class, in ascending order by username.
+     * @return a list of all Users
+     * @since 1.0.0
+     */
+    public List<User> getAllUsers() {
+        final Query<User> query = pm.newQuery(User.class);
+        query.setOrdering("username asc");
+        return executeAndCloseList(query);
+    }
```

Comment:
Please use [`execute`](https://github.com/DependencyTrack/hyades-apiserver/blob/db00ad60729c0d88e34ab856a62a0459d2aa648b/alpine/alpine-infra/src/main/java/alpine/persistence/AbstractAlpineQueryManager.java#L156-L162) here to make this query support pagination. Since large instances will likely have lots of users, we cannot return them all at once.